src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryConfiguration.java

```diff
@@ -58,6 +70,15 @@ public class LibraryConfiguration extends AbstractDescribableImpl<LibraryConfigu
     private String defaultVersion;
     private boolean implicit;
     private boolean allowVersionOverride = true;
+    private boolean allowVersionEnvvar = false;
+    private boolean allowBRANCH_NAME = false;
+    private boolean allowBRANCH_NAME_PR = false;
```

Comment:
Not conventional property naming format.

Reply:
Commented earlier - this one was derived from resolution of `BRANCH_NAME` envvar (or later the extended concept where envvar does not exist so we study SCM context of the build), so retained its spelling. Suggestions welcome, just don't want to make 10 more commits to change it to this or that across the board :)